lib/sinon/assert.js

```diff
@@ -12,8 +6,12 @@
  *
  * Copyright (c) 2010-2013 Christian Johansen
  */
-(function (sinonGlobal, global) {
-    "use strict";
+"use strict";
+(function () {
```

Comment:
It's totally unnecessary but I wanted this PR to clearly show that the actual code didn't change. A next pass can get rid of them.